.babelrc

```diff
@@ -1,7 +1,8 @@
 {
   "presets": [],
   "plugins": [
-    "transform-react-jsx"
+    "transform-react-jsx",
+    "transform-react-stateless-component-name"
```

Comment:
This was required so that the stateless component names (for components from the shared package) appear in tests.